database/schema.sql

```diff
@@ -5,3 +5,57 @@ set client_min_messages to warning;
 drop schema "public" cascade;
 
 create schema "public";
+
+CREATE TABLE "public"."User" (
+	"username" TEXT NOT NULL,
+	"email" TEXT NOT NULL,
+	"userId" serial NOT NULL,
+	CONSTRAINT "User_pk" PRIMARY KEY ("userId")
+) WITH (
+  OIDS=FALSE
+);
+
+
+
+CREATE TABLE "public"."Cart Items" (
+	"price" DECIMAL NOT NULL,
+	"productId" integer NOT NULL,
+	"cartId" integer NOT NULL,
+	CONSTRAINT "Cart Items_pk" PRIMARY KEY ("productId","cartId")
+) WITH (
+  OIDS=FALSE
+);
+
+
+
+CREATE TABLE "public"."Catalog" (
+	"itemType" TEXT NOT NULL,
+	"itemName" TEXT NOT NULL,
+	"itemImage" TEXT NOT NULL,
+	"price" DECIMAL NOT NULL,
+	"productId" serial NOT NULL,
+	CONSTRAINT "Catalog_pk" PRIMARY KEY ("productId")
+) WITH (
+  OIDS=FALSE
+);
+
+
+
+CREATE TABLE "public"."Cart" (
+	"cartId" serial NOT NULL,
+	"createdDate" DATE NOT NULL,
+	"totalPrice" DECIMAL NOT NULL,
+	"userId" serial NOT NULL,
+	CONSTRAINT "Cart_pk" PRIMARY KEY ("cartId")
+) WITH (
+  OIDS=FALSE
+);
+
+
+
+
+ALTER TABLE "Cart Items" ADD CONSTRAINT "Cart Items_fk0" FOREIGN KEY ("productId") REFERENCES "Catalog"("productId");
```

Comment:
You'll need to remove the space in these statements, also. (Two on line 57, two on line 58)